src/components/ChannelComponent.vue

```diff
@@ -41,78 +44,134 @@
     { id: 27, text: 'Haha, dobre teda, dohodneme sa po piatku.', userName: 'Tomas',profilePic: 'https://cdn.quasar.dev/img/avatar1.jpg', timestamp: new Date('2024-03-01T09:30:00') },
     { id: 28, text: 'Platí! Maj sa zatiaľ!', userName: 'Matej',profilePic:'https://cdn.quasar.dev/img/boy-avatar.png', timestamp: new Date('2024-03-01T09:32:00') },
     { id: 29, text: 'Čaukooo', userName: 'Tomas',profilePic: 'https://cdn.quasar.dev/img/avatar1.jpg', timestamp: new Date('2024-03-01T09:35:00') },
+  ])
 
+  const users = ref([
+    { userName: 'Matej', profilePic: 'https://cdn.quasar.dev/img/boy-avatar.png' },
+    { userName: 'Tomas', profilePic: 'https://cdn.quasar.dev/img/avatar1.jpg' },
   ])
 
+  function commandsCheck(message: string) {
+    const messageClean = message.replace(/<[^>]*>/g, '').trim();
+
+    if (messageClean === '/list') {
+      const userList = users.value.map(user => user.userName).join(', ');
+      const listMessage = `Users in the channel: ${userList}`;
```

Comment:
toto mozno zmenit nech to neni message, ked to bude real

Reply:
jo jo toto je len docasne riesenie pre FE